packages/e2e-tests/src/jest.config.js

```diff
@@ -44,10 +44,8 @@ export default {
   setupFilesAfterEnv: [
     'jest-extended/all',
     '<rootDir>/packages/e2e-tests/src/config/bootstrap.js',
-    '@wordpress/jest-puppeteer-axe',
     '@wordpress/jest-console',
     'expect-puppeteer',
   ],
   modulePathIgnorePatterns: ['<rootDir>/build', '<rootDir>/vendor'],
-  reporters: [['jest-silent-reporter', { useDots: true, showWarnings: true }]],
```

Comment:
Removed so we can see more detailed output in CI

Reply:
So is the default output more useful than the dots for the e2e tests?

Reply:
Was finding it helpful to watch the tests in the CI runs to see where things we're hanging i.e. would get to text x or y and than hang vs the .... can swap back but that's my reasoning.